tools/appveyor.psm1

```diff
@@ -108,6 +108,20 @@ function Invoke-AppVeyorInstall
         Update-AppveyorBuild -message $buildName
     }
 
+    # Generate credentials for appveyor remoting tests.
```

Comment:
But we have a race condition - you're changing the machine credentials. If I try to log in after you've changed them, I have no idea what credentials are in use.